src/corefx/System.Globalization.Native/icushim.h

```diff
@@ -15,7 +15,7 @@
 
 // All ICU headers need to be included here so that all function prototypes are
 // available before the function pointers are declared below.
-#include <unicode/locid.h>
+//#include <unicode/locid.h>
```

Comment:
If it is not needed - delete it.

Reply:
Will do, this is a C++ header, based on http://icu-project.org/apiref/icu4c/files.html